build.gradle

```diff
@@ -15,6 +18,9 @@ allprojects {
     repositories {
         google()
         mavenCentral()
+        maven {
+            url 'https://developer.huawei.com/repo/'
```

Comment:
is this optional / controllable at build time by external developers?

Reply:
I made the Huawei dependency `compileOnly`. Let me look into whether I can make the Huawei Maven repo dependency optional as well.

Reply:
So it turns out that repositories aren't included downstream (unlike dependencies, they're not in the pom). So this should be fine.